fhir-server/src/test/java/com/ibm/fhir/server/test/ReferenceMappingVistorTest.java

```diff
@@ -33,64 +32,71 @@
 import com.ibm.fhir.model.type.Meta;
 import com.ibm.fhir.model.type.Reference;
 import com.ibm.fhir.model.type.String;
-import com.ibm.fhir.server.resources.FHIRResource;
 import com.ibm.fhir.server.util.ReferenceMappingVisitor;
 
 public class ReferenceMappingVistorTest {
+    public boolean DEBUG = false;
+
     private Patient patient;
-    private HashMap<java.lang.String , java.lang.String> localRefMap;
+    private HashMap<java.lang.String, java.lang.String> localRefMap;
+
     @BeforeClass
     public void setUp() throws Exception {
         java.lang.String id = UUID.randomUUID().toString();
-        
-        Meta meta = Meta.builder().versionId(Id.of("1"))
-                .lastUpdated(Instant.now(ZoneOffset.UTC))
-                .build();
-        
-        String given = String.builder().value("John")
-                .extension(Extension.builder()
-                    .url("http://www.ibm.com/someExtension")    
-                    .value(String.of("value and extension"))
-                    .build())
-                .build();
-        
-        String otherGiven = String.builder()
-                .extension(Extension.builder()
-                    .url("http://www.ibm.com/someExtension")    
-                    .value(String.of("extension only"))
-                    .build())
-                .build();
-        
-        HumanName name = HumanName.builder()
-                .id("someId")
-                .given(given)
-                .given(otherGiven)
-                .given(String.of("value no extension"))
-                .family(String.of("Doe"))
-                .build();
-        
+
+        Meta meta =
```

Comment:
the formatting changes here don't seem to help much.  i assume that was just the auto-formatter

Reply:
I updated the style of the content.